index.html

```diff
@@ -4,20 +4,21 @@
 	<title>Monopoly</title>
 	<meta http-equiv="Content-type" content="text/html;charset=UTF-8" />
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-	<script type="text/javascript" src="classicedition.js"></script>
+	<script type="text/javascript" src="http://isaiahpatton.github.io/monopoly/classicedition.js"></script>
 	<!-- Comment out the previous line and uncomment the next one to enable the New York City Edition -->
 	<!-- <script type="text/javascript" src="newyorkcityedition.js"></script> -->
-	<script type="text/javascript" src="ai.js"></script>
-	<script type="text/javascript" src="monopoly.js"></script>
-	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<script type="text/javascript" src="//isaiahpatton.github.io/monopoly/ai.js"></script>
+	<script type="text/javascript" src="//isaiahpatton.github.io/monopoly/monopoly.js"></script>
+	<link rel="stylesheet" type="text/css" href="//isaiahpatton.github.io/monopoly/styles.css">
+	<CENTER><img src="https://raw.githubusercontent.com/IsaiahPatton/monopoly/gh-pages/images/mlogo.png"></CENTER>
```

Comment:
Also this image should be in the `body` not the head, and `<center>` is deprecated and should not be used anymore: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center - use a CSS alternative